packages/web/src/utils/withData.tsx

```diff
@@ -48,14 +52,17 @@ export default App => {
 				try {
 					// Run all GraphQL queries
 					const app = (
-						<ApolloProvider client={apollo}>
-							<App
-								{...appProps}
-								Component={Component}
-								router={router}
-								apolloClient={apollo}
-							/>
-						</ApolloProvider>
+						// @ts-ignore
+						<ApolloProviderHoc client={apollo}>
+							<ApolloProvider client={apollo}>
```

Comment:
I think we don't need this inner ApolloProvider yet because we're not using hooks anywhere in the code base.